pyaedt/edb.py

```diff
@@ -206,6 +217,7 @@ def _init_objects(self):
         self._components = Components(self)
         self._stackup = EdbStackup(self)
         self._padstack = EdbPadstacks(self)
+        self._padstack_instance = EdbPadstackInstances(self)
```

Comment:
@ring630 I am assuming you meant `self._padstack_instances` because `self._padstack_instance` is already created and 20 lignes above.
```suggestion
        self._padstack_instance = EdbPadstackInstances(self)
```
```suggestion
        self._padstack_instances = EdbPadstackInstances(self)
```

Reply:
self._padstack is defined twice. I am not sure why, but I am follow the same format. Does self._padstack should be changed to self._padstacks as well?

Reply:
I think it would have to be aligned with the refactoring mentioned by @maxcapodi78 .

Reply:
Now properties and methods in EdbEdbPadstackInstances is merged EdbPadstacks as suggested by  @maxcapodi78 .